examples/30_extended/task_manual_iteration_tutorial.py

```diff
@@ -0,0 +1,84 @@
+"""
+Tasks: retrieving splits
+========================
+
+Tasks define a target and a train/test split. Normally, they are the input to the function
+``openml.runs.run_model_on_task`` which automatically runs the model on all splits of the task.
+However, sometimes it is necessary to manually split a dataset to perform experiments outside of
+the functions provided by OpenML. One such example is in the benchmark library
+`HPOlib2 <https://github.com/automl/hpolib2>`_ which extensively uses data from OpenML,
+but not OpenML's functionality to conduct runs.
+"""
+
+import openml
+
+####################################################################################################
+task_id = 233
```

Comment:
maybe add small comment which task this is and why (few observations, well-known iris, ... ?)

Reply:
Great idea, thanks I lot, I just added that explanation.